### PR TITLE
0.7.5 release

### PR DESCRIPTION
**Pull request by pd0wm**

### CODE REVIEW

**Review by andyh2 — APPROVED** [review]
LGTM. Panda safety tests are so much more readable 👍

**Comment by pd0wm** [comment]
All thanks to @quillford !

File: README.md [hunk 1]
```diff
@@ -57,106 +57,123 @@ openpilot should preserve all other vehicle's stock features, including, but are
...
-| Kia       | Sorento 2018<sup>1</sup>      | All               | Stock            | 0mph               | 0mph              |
-| Kia       | Stinger 2018<sup>1</sup>      | SCC + LKAS        | Stock            | 0mph               | 0mph              |
-| Lexus     | CT Hybrid 2017-18             | All               | Stock<sup>5</sup>| 0mph               | 0mph              |
+| Honda     | Odyssey 2018-20               | Honda Sensing     | openpilot        | 25mph<sup>1</sup>  | 0mph              |
+| Honda     | Passport 2019                 | All               | openpilot        | 25mph<sup>1</sup>  | 12mph             |
+| Honda     | Pilot 2016-18                 | Honda Sensing     | openpilot        | 25mph<sup>1</sup>  | 12mph             |
+| Honda     | Pilot 2019                    | All               | openpilot        | 25mph<sup>1</sup>  | 12mph             |
+| Honda     | Ridgeline 2017-20             | Honda Sensing     | openpilot        | 25mph<sup>1</sup>  | 12mph             |
+| Hyundai   | Sonata 2020                   | All               | Stock            | 0mph               | 0mph              |
```
Comment (geohot) [inline README.md:84]: These shouldn't be here until we sell a harness for them.

Comment (pd0wm) [reply README.md:84]: Oops, though those were put in the community maintained section. We also need to do FW queries, and some final tuning before we promote them to official support.

Will fix before shipping

Comment (pd0wm) [reply README.md]: Fixed

**Comment by geohot** [comment]
This should fix the LGTM analysis: https://github.com/commaai/openpilot/commit/bb9ca1ebd67f3f0d737f1171231cdbfd39c10b51

File: selfdrive/ui/sound.cc [hunk 2]
```diff
@@ -13,28 +13,34 @@ extern "C"{
 #include "slplay.h"
 }
 
+int last_volume = 0;
+
 void set_volume(int volume) {
-  char volume_change_cmd[64];
-  sprintf(volume_change_cmd, "service call audio 3 i32 3 i32 %d i32 1 &", volume);
+  if (last_volume != volume) {
```
Comment (adeebshihadeh) [inline selfdrive/ui/sound.cc:19]: are we sure we're the only ones changing the volume? do the volume buttons do anything?

Comment (adeebshihadeh) [reply selfdrive/ui/sound.cc:19]: Tested on an EON, volume buttons have no effect

**Comment by geohot** [comment]
Can we disable locationd on this branch? It isn't used yet (confirm?), and we don't have the CPU for shadow mode.

Currently at 81% looking at the corolla, w/o locationd we'd be back at 73%. I don't like releases above 75%.

**Comment by pd0wm** [comment]
Updated PR with those changes

@geohot LGTM is still failing

**Comment by geohot** [comment]
Huh, now LGTM fails because Devel can't be built on x86.

**Comment by geohot** [comment]
I'm still working on the LGTM thing. The JavaScript X is a bug, and you seem to have to contact them to disable JavaScript for your project.

The C++ X is because we don't include the external/bin dir with capnp. Moving to capnp 0.7.0 will fix this, and we should be able to use the system capnp with capnpc-c gone.

**Pull request merged**